python/hail/methods/tests.py

```diff
@@ -123,6 +123,10 @@ def test_impute_sex_same_as_plink(self):
 
         self.assertTrue(plink_sex._same(sex, tolerance=1e-3))
 
+        ds = ds.annotate_rows(aaf=(agg.call_stats(ds.GT, ds.v)).AF[0])
```

Comment:
same -- AF[1]. I double checked and the answer won't change if you use AF[0] or AF[1] because it's using `p(1-p)`. However, I'd prefer to be consistent if the parameter name is alternate allele frequency.